PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -338,6 +333,15 @@ def miniAOD_customizeCommon(process):
     for idmod in photon_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDPhotonSelection,None,False,task)
 
+    process.patPhotons.photonIDSources=cms.PSet()
+    for egid in process.egmPhotonIDs.physicsObjectIDs:
+        setattr(process.patPhotons.photonIDSources,egid.idDefinition.idName.value(),cms.InputTag('egmPhotonIDs:'+egid.idDefinition.idName.value()))
+
+    #add the cut base IDs bitmaps of which cuts passed
+    from RecoEgamma.EgammaTools.egammaObjectModifications_tools import makeVIDBitsModifier
+    egamma_modifications.append(makeVIDBitsModifier(process,"egmGsfElectronIDs","egmPhotonIDs"))
```

Comment:
this apparently also can stay in PhysicsTools/SelectorUtils/python/tools/vid_id_tools.py perhaps even as a part of setupAllVIDIdsInModule

Reply:
missed that it was already been done there, can just delete the IDsources part for  electrons & photons. The object modifications, those have to stay where they are.